asreview/entry_points/simulate.py

```diff
@@ -171,7 +171,7 @@ def _simulate_parser(prog="simulate", description=DESCRIPTION_SIMULATE):
              f"Default {DEFAULT_N_INSTANCES}.")
     parser.add_argument(
         "--n_queries",
-        type=int,
+        type=str,
```

Comment:
I think it might be better to define a custom type over here that accepts integers and "min". Like described here: https://stackoverflow.com/questions/53445769/argparse-define-custom-actions-or-types-with-additional-arguments.